app/backend/src/couchers/models.py

```diff
@@ -653,3 +653,262 @@ class InitiatedUpload(Base):
     @hybrid_property
     def is_valid(self):
         return (self.created <= func.now()) & (self.expiry >= func.now())
+
+
+class Node(Base):
+    """
+    Node, i.e. geographical subdivision of the world
+    """
+
+    __tablename__ = "nodes"
+
+    id = Column(BigInteger, primary_key=True)
+
+    name = Column(String, nullable=False)
+    parent_node = Column(BigInteger, ForeignKey("nodes.id"))
+    polygon = Column(Geometry(geometry_type="POLYGON", srid=4326))
+    official_cluster_id = Column(BigInteger, ForeignKey("clusters.id"), nullable=False, unique=True)
```

Comment:
These kinds of FKs don't need the type, and so we shouldn't repeat it (I know it's inconsistent in the rest of models), e.g., this can be changed to

```py3
official_cluster_id = Column(ForeignKey("clusters.id"), nullable=False, unique=True)
```

Reply:
All fixed